### PR TITLE
2238 2239 arithmetic

### PR DESCRIPTION
**Pull request by mcmajam**
### Changelog category (leave one):
- New Feature

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Added Kusto functions abs(), ceiling(), exp(), exp10(), exp2(), log(), log10(), log2(), pow(), and sqrt()

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

**Review by ltrk2 — CHANGES_REQUESTED** [review]
I think you could have a few cases of invalid input types (eg. `print sqrt('asd')`) to see that error handling is identical.

Lastly, please don't forget to format your code using `clang-format`.

File: src/Parsers/Kusto/KQL_ReleaseNote.md [hunk 1]
```diff
@@ -5,6 +5,26 @@
...
+- [log()](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/log-function)
+   `print log(5)`
+- [log2()](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/log2-function)
+   `print log2(5)`
+- [log10()](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/log10-function)
+   `print log10(5)`
+- [pow()](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/powfunction)
+   `print pow(2, 3)`
+- [sqrt()](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/sqrtfunction) 
+   `print sqrt(256)`
```
Comment (ltrk2) [inline src/Parsers/Kusto/KQL_ReleaseNote.md:27]: Please create a new heading for the February release and move all this under that.

File: tests/queries/0_stateless/02366_kql_func_math.sql [hunk 2]
```diff
@@ -5,3 +5,25 @@ print isnan(4.2);
...
+print '-- log --';
+print log(5);
+print '-- log2 --';
+print log2(5);
+print '-- log10 --';
+print log10(5);
+print '-- pow --';
+print pow(2, 3);
+print '-- sqrt --';
+print sqrt(256);
```
Comment (ltrk2) [inline tests/queries/0_stateless/02366_kql_func_math.sql:29]: Please add any edge cases you've tried (eg. `sqrt(-1)`), even those that throw exceptions (eg. `abs('asd')`).

**Review by ltrk2 — CHANGES_REQUESTED** [review]
